test/src/main/java/org/apache/accumulo/test/functional/ManyWriteAheadLogsIT.java

```diff
@@ -71,6 +77,38 @@ protected int defaultTimeoutSeconds() {
     return 10 * 60;
   }
 
+  @Before
+  public void alterConfig() throws Exception {
+    if (getClusterType() == ClusterType.MINI) {
+      return;
+    }
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      InstanceOperations iops = client.instanceOperations();
+      Map<String,String> conf = iops.getSystemConfiguration();
+      majcDelay = conf.get(Property.TSERV_MAJC_DELAY.getKey());
+      walogSize = conf.get(Property.TSERV_WALOG_MAX_SIZE.getKey());
+
+      iops.setProperty(Property.TSERV_MAJC_DELAY.getKey(), "1");
+      iops.setProperty(Property.TSERV_WALOG_MAX_SIZE.getKey(), "1M");
+
+      getClusterControl().stopAllServers(ServerType.TABLET_SERVER);
+      getClusterControl().startAllServers(ServerType.TABLET_SERVER);
+    }
+  }
```

Comment:
Could you just put this in configureMiniCluster()?  Or does it have to be before configureMiniCluster is called?

Reply:
This is an interesting solution for the standalone case. However, if it is necessary to alter the configuration of the system for the test, then it is probably not suitable for a standalone test, and should be recategorized so as to be skipped when running against a standalone instance.

Reply:
@milleruntime configureMiniCluster() is called when a MAC is used.  In this case we are running standalone so it would not be called.